.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+
+env:
+  REACT_APP_NETWORK_URL: 'https://mainnet.infura.io/v3/4bf032f2d38a4ed6bb975b80d6340847'
+  APP_ID: 1
+
+jobs:
+  integration-tests:
+    name: Cypress
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      # Install deps only
+      - name: Cypress install deps
+        uses: cypress-io/github-action@v2
+        with:
+          runTests: false
+
+      - name: Install Cypress binary
+        if: failure()
+        run: npx cypress install
```

Comment:
won't this install `@latest` by default? :thinking:

Reply:
My understanding is that, it looks to the version you have in your package.json

Reply:
Anxo is right